### PR TITLE
UI library post

### PR DESCRIPTION
**Pull request by andrew-secret**

### CODE REVIEW

File: content/posts/2021-03-08-why-you-should-use-a-ui-library/2021-03-08-why-you-should-use-a-ui-library.mdx [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: Why I should use a UI library?
+date: '2020-05-17T13:37:03.284Z'
```
Comment (HenrikFricke) [inline content/posts/2021-03-08-why-you-should-use-a-ui-library/2021-03-08-why-you-should-use-a-ui-library.mdx:3]: You probably have it on your list, but we have to update the date here. Also, the file and folder needs to be updated 😇

File: content/posts/2021-03-08-why-you-should-use-a-ui-library/2021-03-08-why-you-should-use-a-ui-library.mdx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+author: andre
+---
+
+In this post, I want to point out what advantages you can gain from an
+open-source UI library and how it can make your life easier.
+
+## Material Design Tab Example
+
+A while ago I decided to rebuild a React Tab component from Material-UI with
```
Comment (HenrikFricke) [inline content/posts/2021-03-08-why-you-should-use-a-ui-library/2021-03-08-why-you-should-use-a-ui-library.mdx:12]: Maybe we could add a link here to the Material UI docs?

File: content/posts/2021-03-08-why-you-should-use-a-ui-library/2021-03-08-why-you-should-use-a-ui-library.mdx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+Vanilla Javascript to get an overview of the complexity and the effort. My main
+goal there was to focus on accessibility like navigation with arrow keys tab
```
Comment (HenrikFricke) [inline content/posts/2021-03-08-why-you-should-use-a-ui-library/2021-03-08-why-you-should-use-a-ui-library.mdx:14]: "…like navigating with arrows keys or tab index" - does it sound better?

File: content/posts/2021-03-08-why-you-should-use-a-ui-library/2021-03-08-why-you-should-use-a-ui-library.mdx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+You don't have to write each time the same code when you start a new project. I
+often underestimate the complexity of a seemingly easy UI component. In most
+cases, I decide to use an open-source library. The main advantages for me are:
+
+- don't have to write and maintain complex javascript logic
+- fewer bugs in a project
+- Accessibility is taken into account
+- UI is way more solid and covers edge cases for me
+
```
Comment (HenrikFricke) [inline content/posts/2021-03-08-why-you-should-use-a-ui-library/2021-03-08-why-you-should-use-a-ui-library.mdx:61]: Could be also interesting: The power of the open-source community. Hundreds of people polishing components, finding bugs and fixing them. Is the UI component really your core business or should you rather focus on the problem you really wanna solve.

Comment (HenrikFricke) [reply content/posts/2021-03-08-why-you-should-use-a-ui-library/2021-03-08-why-you-should-use-a-ui-library.mdx:61]: You basically mentioned all the points, maybe it's just cool to point out the strengths of open-source communities.

**Review by HenrikFricke — COMMENTED** [review]
Really cool, just a few thoughts :)

File: content/posts/2021-03-08-why-you-should-use-a-ui-library/2021-03-08-why-you-should-use-a-ui-library.mdx (hunk 4 above)
Comment (andrew-secret) [reply content/posts/2021-03-08-why-you-should-use-a-ui-library/2021-03-08-why-you-should-use-a-ui-library.mdx:61]: Totally agree

**Pull request merged**